python-sdk/tests/files/locations/test_google_drive.py

```diff
@@ -0,0 +1,95 @@
+from unittest.mock import patch
+
+import pytest
+from airflow.providers.google.suite.hooks.drive import GoogleDriveHook
+from googleapiclient.discovery import Resource
+
+from astro.files.locations import create_file_location
+
+
+def test_get_transport_params_for_gdrive():  # skipcq: PYL-W0612, PTC-W0065
+    """test get_transport_params() method which should return gdrive resource client"""
+    path = "gdrive://bucket/some-file"
+    location = create_file_location(path)
+    credentials = location.transport_params
+    assert isinstance(credentials["client"], Resource)
+
+
+@patch("airflow.providers.google.suite.hooks.drive.GoogleDriveHook.get_conn")
+@patch("astro.files.locations.google.gdrive._find_item_id")
+def test_remote_object(
```

Comment:
what does `test_remote_object` mean? Make it a bit more descriptive please

Reply:
Changed it